File: NModM/nmodm.rb
# N mod M soln in rb for codeeval.com by steven a dunn

File.open(ARGV[0], 'r').each_line do |line|
    line = line.chomp.split(',')
    n = line[0].to_i
    m = line[1].to_i

    x = n / m
    y = x * m
    mod = n - y

    puts mod
end

File: NModM/c#/NModM.cs
// n mod m soln in c# for code eval by steven a dunn

using System;
using System.IO;

class NModM {
  public static void Main(string[] args) {
    using (StreamReader reader = new StreamReader(args[0])) {
      while (!reader.EndOfStream) {
        string line = reader.ReadLine();
        if (null == line)
          continue;

        string[] inputs = line.Split(',');
        int n = Int32.Parse(inputs[0]);
        int m = Int32.Parse(inputs[1]);

        int x = n / m;
        int y = x * m;
        int mod = n - y;

        Console.WriteLine(mod);
      }
    }
  }

}